### ISSUE TITLE
ModuleNotFoundError after built from source

### ISSUE THREAD

**Issue by chtsai0105** [GCS-ZHN/diamond4py#6]
Hi - I'm interested in adding/modifying some of the functions in this package so I just tried to build from source. But an error occurred while running `import diamond4py` after install by `pip install -v .` under the source folder. I've checked the `diamond4py` folder under `site-package` and seems like the `libdiamond.cpython-312-x86_64-linux-gnu.so` is successfully built. I also checked by `ldd [path-of-libdiamond.cpython-312-x86_64-linux-gnu.so]` and confirmed that all the required binaries are exists.

I've also tried to download the 0.0.6 release but still having the same problem. However, if I directly install by `pip install diamond4py` the issue is resolved. Yet I'm not very familiar what the differences between building from pypi and from source so still unable to resolve this. Can you please test the building step again to see if you can replicate the problem?

**Comment by chtsai0105** [comment]
I finally resolved the issue by changing the working directory out from the source folder...

If I ran python in the source folder and import the library I got a `ModuleNotFoundError`. But if I changed the working directory to somewhere else and import the library again the issue is no longer persist.
I guess the issue somewhat related to the path - since the source folder have exactly the same name as the module folder inside the python `site-package` and for some reason it was looking for the module in the source folder when doing python import there.
